packages/core/fluxcd/patches/hostNetwork.diff

```diff
@@ -0,0 +1,52 @@
+diff --git a/charts/flux-operator/README.md b/charts/flux-operator/README.md
+index 071e024..2fccc5f 100644
+--- a/charts/flux-operator/README.md
++++ b/charts/flux-operator/README.md
+@@ -36,6 +36,7 @@ see the Flux Operator [documentation](https://fluxcd.control-plane.io/operator/)
+ | commonAnnotations | object | `{}` | Common annotations to add to all deployed objects including pods. |
+ | commonLabels | object | `{}` | Common labels to add to all deployed objects including pods. |
+ | fullnameOverride | string | `""` |  |
++| hostNetwork | bool | `false` | If `true`, start flux-operator in hostNetwork mode. |
```

Comment:
This is not in sync with the upstream chart, please copy https://github.com/controlplaneio-fluxcd/charts/tree/main/charts/flux-operator

Reply:
We have `make update` which pulls from upstream, and a new PR for flux-operator 0.6 - these changes will get synced back before they are released. I already updated one of the patches to remove fuzz, (and deleted the other one for hostNetwork that is no longer needed because you merged it.)